_test/test_sqw_class/test_eval.m

```diff
@@ -119,10 +105,14 @@ function test_sqw_eval_no_pix(obj)
             end
             assertFalse(failed,err_message);
 
+            assertEqual(size(ds.data.s), size(sqw_nopix.data.s));
+            % the first value is calculated, it doesn't matter what
+            % the actual value is for the purpose of this test
```

Comment:
The comment sounds as if you are testing that something got produced rather than what its value is. The test function entering sqw_eval will produce output equivalent to expected. But what you mean by the actual words of the comment is not clear to me.